### PR TITLE
Added TotalReadoutTime for subject's diffusion data

### PR DESCRIPTION
**Pull request by slimnsour**
Adding TotalReadoutTime so functions like `eddy_openmp` can be run for the diffusion data (see https://github.com/nipreps/dmriprep/pull/144#issuecomment-764513887). 0.05 was used as that was the value used in FSL's documentation [here](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/eddy/Faq).

### CODE REVIEW

**Comment by oesteban** [comment]
I had missed this - thanks for the contribution! Could you please copy the comment of your message into the CHANGES file? (basically, add a new entry explaining what you did and why - we want to leave a trail about the decision)

**Comment by slimnsour** [comment]
Good point, done!

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,10 @@
+2.0.0	2021-01-25
+
+	- Added TotalReadoutTime so functions like eddy_openmp can be run for the diffusion 
+	  data (see https://github.com/nipreps/dmriprep/pull/144#issuecomment-764513887). 
+      0.05 was used as that was the value used in FSL's documentation
```
Comment (oesteban) [inline CHANGES]:
```suggestion
        0.05 was used as that was the value used in FSL's documentation 
```

File: CHANGES [hunk 2]
```diff
@@ -1,3 +1,10 @@
+2.0.0	2021-01-25
```
Comment (oesteban) [inline CHANGES]:
```suggestion
2.0.1	2021-01-25
```

**Pull request merged**